apps/antalmanac/src/components/Header/Import.tsx

```diff
@@ -56,8 +57,10 @@ function Import() {
             try {
                 zotcourseImport = await queryZotCourse(zotcourseScheduleName);
             } catch (e) {
-                openSnackbar('error', 'Could not import from Zotcourse.');
-                console.error(e);
+                if (e instanceof Error) {
```

Comment:
issue: generic Error catching may allow unexpected errors to be shown, see below comment in `zotcourse.ts` for suggestion